tests/ci/run_check.py

```diff
@@ -189,17 +172,42 @@ def main():
             pr_info,
         )
         sys.exit(1)
+
+    PR_CHECK = "PR Check"
+
+    if FEATURE_LABEL in pr_info.labels and not pr_info.has_changes_in_documentation():
+        print(
+            f"The '{FEATURE_LABEL}' in the labels, "
+            "but there's no changed documentation"
+        )
+        post_commit_status(
+            commit,
+            "failure",
+            "",
+            f"expect adding docs for {FEATURE_LABEL}",
+            PR_CHECK,
+            pr_info,
+        )
```

Comment:
It still should be above line 162

Reply:
because we set mergeable status there? i added update for mergeable status. is that ok now?